apps/demo-iframe/src/app.css

```
@@ -0,0 +1,224 @@
+  --cc-webplayer-radius-gallery: 0.4rem;
+}
+
+/* Page Styles */
+
+.header {
+  & .header__container {
+    max-width: 1254px;
+    margin-inline: auto;
+    padding: 8px;
+
+    display: flex;
```

Stylelint (17.11.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)